src/main/java/duke/commands/DeleteCommand.java

```diff
@@ -0,0 +1,32 @@
+package duke.commands;
+
+import duke.DukeException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+
+public class DeleteCommand extends Command {
+    private String commands;
+
+    public DeleteCommand(String commands) {
+        this.commands = commands;
+    }
+
+    @Override
+    public String execute(TaskList tasks, Storage storage) throws DukeException {
```

Comment:
For your execute method here, you may want to "make the happy path more prominant" under the code quality section of the SE textbook. This should allow others to understand the exceptions are you checking for much easily and allow them to understand the main code easily.